spring-cloud-data-admin/src/test/java/org/springframework/cloud/data/admin/AdminApplicationTests.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Minor nitpick - this line should be indented 5 spaces in (6 if you include the one space between the `*` and the first character of every other line.